specs/provider.yaml

```diff
@@ -660,6 +660,17 @@ paths:
 
         If the resource's plan matches the given plan then a `200 Success` or
         `204 No Content` response should be returned.
+
+        **Managing Plan Upgrade and Downgrade Options**
+
+        To block a plan change request simply return a `400 Bad Request` response
+        with an error message describing why the change is not allowed. Our system 
+        will handle that and communicate to the user.
```

Comment:
Suggestion: `Our system will handle that and communicate to the user.` -> `Manifold will handle the response and communicate it to the user.`